CONTRIBUTING.md

```diff
@@ -111,14 +107,6 @@ and use the following VSCode user settings:
 ```json
 "editor.formatOnSave": true,
 "rust-analyzer.rustfmt.extraArgs": ["+nightly"],
-"rust-analyzer.check.overrideCommand": [
```

Comment:
Do we not want this?

Reply:
this is likely a bit opinionated, so fine with removing this

Reply:
If you're not running good enough hardware this is a pain